### PR TITLE
APP/IATA for RKSI

### PR DESCRIPTION
**Pull request by kusterjs**
connecting RKSS (APP) to RKSI airport

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -10650,7 +10650,7 @@ RKSD|Maesanri|37.35|127.26639||RKRR|0
 RKSF|Seoul-Yongdungp'O Rokaf Wc|37.5|126.93306||RKRR|0
 RKSG|Pyongtaek Ab|36.96|127.033333||RKRR|0
 RKSH|Command Post Tango|37.51667|126.98306||RKRR|0
-RKSI|Incheon Intl|37.463333|126.440000||RKRR|0
+RKSI|Incheon Intl|37.463333|126.440000|RKSS|RKRR|0
```
Comment (Adrian2k) [inline VATSpy.dat:10653]: There is already have an airport entry for RKSS.

RKSS|Gimpo Intl|37.556944|126.797500||RKRR|0

Comment (kusterjs) [reply VATSpy.dat:10653]: I know, this is just to connect the RKSS approach to the RKSI airport, as it covers this one as well

**Comment by amosngSP** [comment]
Hmm RKSS_APP covers both RKSS and RKSI though, sure to go ahead with this change?

**Comment by prryplatypus** [comment]
Isn't that exactly what this PR is solving @amosngSP? 😄

**Comment by amosngSP** [comment]
> Isn't that exactly what this PR is solving @amosngSP? 😄

Wait, is this to show RKSS_APP on RKSI?
Wait, wouldn't that mess up RKSS_TWR and RKSS_GND too?

**Comment by prryplatypus** [comment]
That is a great question for which I'm not really too sure about the answer. @NelisV @Adrian2k maybe know?

**Comment by kusterjs** [comment]
Shouldn't interfere with TWR/GND stations, as these are displayed related to the airport symbol. But the APP stations are displayed in a circle. But the RKSS circle should also enable toggeling flight to/from RKSI, hence my request.

**Comment by Adrian2k** [comment]
It will break the RKSS airport placement in VATSpy..

image

However if you move the RKSS airport entry so it is above RKSI this does not happen (RKSS is placed at its original position, only 1 approach ring), but adding RKSI does nothing with toggeling inbouds or outbound for RKSI, it simply is not supported to be like that.

You can always toggle the RKSI inbounds/outbounds by right clicking RKSI itself.
image

All in all, it is not possible to create the outcome you are requesting here.
I will keep the PR open incase you think I missed something.

**Pull request closed without merging**